data/liveinst/liveinst

```diff
@@ -21,6 +21,12 @@
 # Allow running another command in the place of anaconda, but in this same
 # environment.  This allows storage testing to make use of all the module
 # loading and lvm control in this file, too.
+if [ "$(id -u)" -ne 0 ]; then
+    xhost +si:localuser:root
+    unset XAUTHORITY
+    exec pkexec "$0" "$@"
+fi
+
 ANACONDA="${LIVECMD:=anaconda --liveinst --graphical}"
```

Comment:
I think the comment belongs to the old code, and the new code could be described as "Always run with root privileges"...?

Reply:
indeed. I pushed some comment changes. let me know if it looks better to you